storage/src/robocorp/storage/__init__.py

```diff
@@ -48,7 +48,27 @@ def list_assets() -> List[str]:
     Returns:
         A list of available assets' names
     """
-    return [asset["name"] for asset in _get_client().list_assets()]
+    client = _get_client()
+    page = client.list_assets()
```

Comment:
More of a thought than anything else, but if the new APIs prefer having pagination in these types of endpoints, the upcoming API client libraries probably should have some nice abstraction for dealing with paginated responses.

Reply:
Yes, that will be a very interesting task for `robocorp.workspace` given the raw Py API clients.